nixos/modules/services/cluster/kubernetes/default.nix

```diff
@@ -339,9 +370,9 @@ in {
         type = types.str;
       };
 
-      admissionControl = mkOption {
+      enableAdmissionPlugins = mkOption {
```

Comment:
What change does this require a user of admissionControl to make? If any, this should be documented in the release notes for 18.09